pkg/models/active_node.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+//ActiveNode model
+type ActiveNode struct {
+	Id        int64  `json:"id"`
+	NodeId    string `json:"node_id"`
+	Heartbeat int64  `json:"heartbeat"`
+	Sequence  int32  `json:"sequence"`
+}
+
+type GetActiveNodeByIDQuery struct {
+	Id     int64
+	Result []*ActiveNode
+}
+
+type SaveActiveNodeCommand struct {
+	ActiveNode []*ActiveNode
```

Comment:
Why do we have any array over here? is there any use case where we want to update all or multiple nodes at a time from a single node (hope not) - @sanchitraizada